README.md

```diff
@@ -39,19 +40,28 @@ send the mail:
     from django_ilmoitin.registry import notifications
     
     notifications.register("event_created", "Event created")
+    notifications.register("event_deleted", "Event deleted")
     ```
 
-4. Create a `dummy_context.py` file in django app and add dummy context data:
+4. Create a `dummy_context.py` file in django app and add dummy context data.
+Either use the codes of notifications that you registered in the previous step, or
+use the const `DEFAULT_CONTEXT` to make some variables available for all templates:
```

Comment:
This could mention about the possibility to use enums as well.

Reply:
Also, is it possible to send notifications using an enum as the type?

Reply:
we do not support passing enums as arguments to `send_notification()` util yet, nor do we support it in the `notifications.register()` method. I would say, we need to improve enum support first, and then add a section, that would show the flow with the enums tothe readme. I am running out of time for this right now. So, let's put it all in the backlog for now?

Reply:
Sure, makes sense.